data/tmpl/values/content5.html

```diff
@@ -34,7 +34,7 @@
         </ul>
         <button id="tell_me" class="button">Show Answer</button>
         <div id="secondary_content" class="secondary-content hide">
-          <p>Love the Web? Get the <a href="https://www.mozilla.org/#newsletter-subscribe" id="secondary_cta" data-intent="newsletter">Mozilla newsletter</a> and help us keep it open and free.</p>
+          <p>Love the Web? Get the <a href="https://www.mozilla.org/#newsletter-subscribe?utm_source=fxonboarding&amp;utm_medium=firefox-browser&amp;utm_campaign=onboardingv1&amp;utm_content=community" id="secondary_cta" data-intent="newsletter">Mozilla newsletter</a> and help us keep it open and free.</p>
```

Comment:
@schalkneethling same as above, reverse the order of the URL. query parameters first and then @newsletter-subscribe